meta-ov/recipes-apps/ovmenu-ng-skripts/ovmenu-ng-skripts_0.1.bb

```diff
@@ -23,6 +23,7 @@ SRC_URI = "\
 	file://update-system.sh \
 	file://download-igc.sh \
 	file://transfer-xcsoar.sh \
+	file://system-info.sh \
```

Comment:
This has nothing to do with rearranging menu items, and probably this fails the build because this script doesn't exist (yet).

Reply:
you are right!